test/plugin/test_out_forward.rb

```diff
@@ -29,9 +32,13 @@ def setup
   ]
 
   def create_driver(conf=CONFIG)
-    d = Fluent::Test::BufferedOutputTestDriver.new(Fluent::ForwardOutput) {
+    Fluent::Test::Driver::Output.new(Fluent::Plugin::ForwardOutput) {
       attr_reader :responses, :exceptions
 
+      def write(chunk)
```

Comment:
`Output#implement?` checks the features of that plugin by checking instance methods implemented in just that class, without methods of superclasses.
This test code defines a one-time class inherits ForwardOutput. If this method is not defined, `Output#implement?` returns false for check of buffered output feature.

It's better to be fixed in future, but currently, this code works fine.